### ISSUE TITLE
Stream Deck + support

### ISSUE THREAD

**Issue by jer78** [GenericMale/streamdeck-sonos#8]
Will this be updated to support the Stream Deck+? Specifically I was looking for a way to control my Sonos volume with the dials. Thanks

**Comment by mgegerfelt** [comment]
Same here, I would love the possibility for Stream Deck+ Dials support. Thank you for the plugin!

**Comment by jeremy-farrance** [comment]
Same here!

**Comment by ghost** [comment]
+1 for this - especially being able to control volume using a knob, and then mute / unmute the speaker using a press on the knob

**Comment by indomitableMan** [comment]
+1 for this!! Volume control (with mute) using a knob would be wonderful

**Comment by indomitableMan** [comment]
I took a look at this code, and I think I have been able to get everything updated to support adjusting the volume and muting the Sonos using a knob on the Stream Deck+. I have created a separate branch and pull request with the updated code, and I need a little assistance with testing out these changes.

If you are interested in helping to test out these changes, please leave a comment here.

**Comment by jer78** [comment]
I finally upgraded to the Stream Deck + and ready to test. Let me know how I can help

**Comment by outbound** [comment]
Happy to test. Please let me know if and when you resume development.

**Comment by jeremy-farrance** [comment]
> I took a look at this code, and I think I have been able to get everything updated to support adjusting the volume and muting the Sonos using a knob on the Stream Deck+. I have created a separate branch and pull request with the updated code, and I need a little assistance with testing out these changes.
> 
> If you are interested in helping to test out these changes, please leave a comment here.

I realize its been a minute, but I would still be interested in this and happy to test or help in some way if I can.

**Comment by markOteams** [comment]
@jeremy-farrance 
Yes please, I am ready to test.   Please let me know what I need to know to help out.  It would be a be help for me.
